nuwebots/worlds/robocup_nuoptim.wbt

```diff
@@ -0,0 +1,66 @@
+#VRML_SIM R2021b utf8
+WorldInfo {
+  info [
+    "Description: walk optimisation world"
+    "Version 0.2"
+  ]
+  title "Robocup V-HL Kid"
+  basicTimeStep 8
+  optimalThreadCount 1
+  physicsDisableTime 0.1
+  physicsDisableLinearThreshold 0.1
+  physicsDisableAngularThreshold 0.1
+  contactProperties [
+    ContactProperties {
+      material1 "grass"
+      coulombFriction [
+        0.5
+      ]
+      softCFM 0.03
+    }
+    ContactProperties {
+      material1 "grass"
+      material2 "robocup soccer ball"
+      coulombFriction [
+        0.5
+      ]
+      bounce 0.76
+      softCFM 0.05
+    }
+    ContactProperties {
+      material2 "robocup soccer ball"
+      bounce 0.76
+    }
+  ]
+}
+Viewpoint {
+  orientation 0.791 -0.345 -0.505 1.42
+  position -12.0 -12.0 4.96
+}
+TexturedBackground {
+  texture "stadium_dry"
+}
+TexturedBackgroundLight {
+  texture "stadium_dry"
+}
+RobocupSoccerField {
+  size "adult"
+}
+DEF RED_PLAYER_1 NUgus {
+  supervisor TRUE
+  controller "nuoptimiser"
+  name "red player 1"
+  translation -7.0 -3.0 0.51  # 0.0 0.0 0.51 
+  rotation 0 0 1 0
+  controllerArgs [
+    "10001"
+    "1",
+    "127.0.1.1",
+    "10.1.0.102"
+    ]
+  }
+#1.571592653589793
+#Viewpoint {
+#  orientation 0.8938498316033444 -0.2697810913064417 -0.35812098698090605 1.4005817359063022
+#  position -8.658964597484593 -10.448717096347492 4.001015029323764
+#}
```

Comment:
removed if commented out?

Reply:
Yes, should remove before merge. The commented out bits are just saving useful constants.
1.571592653589793 is 90 degrees in radians
The Viewpoint defines the starting location of the camera, useful for getting a good starting camera angle on recordings. I don't remember which Viewpoint is which, but both aren't needed, so might as well remove the commented out one.